### PR TITLE
Improve restoration process requirements

### PR DESCRIPTION
**Pull request by Wykiki**
### For Rancher (product) docs only
When contributing to docs, please update the versioned docs. For example, the docs in the v2.6 folder of the `rancher` folder.

Doc versions older than the latest minor version should only be updated to fix inaccuracies or make minor updates as necessary. The majority of new content should be added to the folder for the latest minor version.

### CODE REVIEW

**Comment by jtravee** [comment]
@jiaqiluo could you give a technical review of the above?

See also [#3530](https://github.com/rancher/docs/issues/3530).

File: content/rancher/v2.6/en/backups/migrating-rancher/_index.md [hunk 1]
```diff
@@ -90,11 +90,23 @@ spec:
...
+1. Watch the Restore status:
+    ```
+    kubectl get restore
+    ```
+1. Watch the restoration logs:
+    ```
+    kubectl logs -n cattle-resources-system --tail 100 -f rancher-backup-xxx-xxx
+    ```
+
+Once the Restore resource has the status `Completed`, you can continue the Rancher installation.
```
Comment (jiaqiluo) [inline content/rancher/v2.6/en/backups/migrating-rancher/_index.md:109]:
```suggestion
Watch the Restore status:
    ```
    kubectl get restore
    ```
Watch the restoration logs:
    ```
    kubectl logs -n cattle-resources-system --tail 100 -f rancher-backup-xxx-xxx
    ```

Once the Restore resource has the status `Completed`, you can continue the Rancher installation.
```

Those two steps are applicable for all scenarios, not only when using encryption. So I would suggest moving them to one level higher than the encryption block.

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

**Comment by Wykiki** [comment]
I think I've updated the Markdown the correct way, but won't have time to test it locally until week-end.

**Review by jiaqiluo — APPROVED** [review]
LGTM.

**Comment by btat** [comment]
Thank you for the contribution. The code blocks weren't rendering so I went ahead and fixed that.

**Pull request merged**